### PR TITLE
add translation to "About" section

### PR DESCRIPTION
**Pull request by Korben85**
add translation to "About" section

### CODE REVIEW

File: app/scripts/dimApp.i18n.js [hunk 1]
```diff
@@ -13,6 +13,25 @@
           General: "General",
           Weapons: "Weapons",
           Vault: "Vault",
+          About: {
+            About: "About Destiny Item Manager (DIM)",
```
Comment (delphiactual) [inline app/scripts/dimApp.i18n.js]: This needs to be changed to AboutDIM: ""About Destiny Item Manager (DIM)",

File: app/scripts/dimApp.i18n.js [hunk 2]
```diff
@@ -238,6 +257,25 @@
           Armor: "Rüstung",
           General: "Allgemein",
           Vault: "Tresor",
+          About: {
+            About: "Über Destiny Item Manager (DIM)",
```
Comment (delphiactual) [inline app/scripts/dimApp.i18n.js]: AboutDIM:

File: app/views/about.html [hunk 3]
```diff
@@ -1,65 +1,64 @@
 <div>
-  <h1>About Destiny Item Manager (DIM)</h1>
+  <h1><span translate="About.About"></span></h1>
```
Comment (delphiactual) [inline app/views/about.html]: About.AboutDIM

**Comment by Korben85** [comment]
Fixed that. Thanks for pointing out.

delphiactual <notifications@github.com> schrieb am Mo., 31. Okt. 2016 um
17:38 Uhr:

**Comment by bhollis** [comment]
This is great! I'd prefer if we could figure out a way to just translate the whole HTML page rather than relying on a bunch of strings, though.

**Comment by delphiactual** [comment]
@bhollis ng-show based on language?

**Comment by bhollis** [comment]
I'd prefer if we loaded different templates per language.

**Comment by delphiactual** [comment]
@Korben85 I used your translations to make the about.html for german. If you want you can do the support.html

https://github.com/DestinyItemManager/DIM/pull/1125

**Comment by bhollis** [comment]
Closing in favor of #1125.

**Comment by Korben85** [comment]
Yeah, that's a good idea!
I'll take a look on the support page.

**Pull request closed without merging**